modules/op-custom-parameter-expansion.adoc

```diff
@@ -0,0 +1,74 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: CONCEPT
+[id="op-custom-parameter-expansion_{context}"]
+= Custom parameter expansion
+
+You can use {pac} to expand a custom parameter within your `PipelineRun` resource by using the `params` field. You can specify a value for the custom parameter inside the template of the `Repository` custom resource (CR). The specified value replaces the custom parameter in your pipeline run.
+
+[NOTE]
+====
+Use the custom parameter expansion feature only in specific scenarios when you cannot use the Tekton `PipelineRun` parameters. This is preferred because Tekton parameters are defined in a `Pipeline` resource and customized alongside it inside a Git repository, whereas custom parameters are defined and customized where the `Repository` CR is located. So, you cannot manage your CI/CD pipeline from a single point. 
```

Comment:
Can you please rephrase the second sentence to get rid of the passive "this is preferred"?

Reply:
Nit: I'd additionally try to get rid of "whereas", maybe by splitting the sentence into two: 
"(...) Tekton parameters are defined in a `Pipeline` resource and customized alongside it inside a Git repository. However, custom parameters are defined and customized where the `Repository` CR is located."